### PR TITLE
Add docs for Plaid Wallet Onboard

### PR DESCRIPTION
**Pull request by clayallsopp**

### CODE REVIEW

File: docs/develop/wallets/plaid-wallet-onboard.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+id: plaid-wallet-onboard
+title: Plaid Wallet Onboard
```
Comment (ashucoder9) [inline docs/develop/wallets/plaid-wallet-onboard.md:3]:
```suggestion
sidebar_label: Plaid Wallet Onboard
```

File: docs/develop/wallets/plaid-wallet-onboard.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+description: Easily connect your dApp to hundreds of self-custody wallets.
```
Comment (ashucoder9) [inline docs/develop/wallets/plaid-wallet-onboard.md:4]:
```suggestion
description: Wallet Onboard allows your app to connect to hundreds of self-custody wallets, including browser plugin wallets, WalletConnect-compatible mobile wallets, and custom protocols or UX required by hybrid and hardware wallets.
```

Comment (ashucoder9) [inline docs/develop/wallets/plaid-wallet-onboard.md:4]:
```suggestion
description: Wallet Onboard allows you to easily connect your dApp to hundreds of self-custody wallets.
```

File: docs/develop/wallets/plaid-wallet-onboard.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  - wallet
+  - dapp
+  - integrate
+  - guide
+image: https://wiki.polygon.technology/img/polygon-wiki.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+**Plaid Wallet Onboard** lets you integrate multiple wallets such as Coinbase Wallet, Metamask, and more into your dApp.
```
Comment (ashucoder9) [inline docs/develop/wallets/plaid-wallet-onboard.md:17]:
```suggestion
**Plaid Wallet Onboard** allows your dApp to connect to hundreds of self-custody wallets, including browser plugin wallets, WalletConnect-compatible mobile wallets, and custom protocols or UX required by hybrid and hardware wallets. It's easier than ever to integrate Metamask, Coinbase Wallet, Rainbow, and more into your dApp.
```

File: docs/develop/wallets/plaid-wallet-onboard.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+Wallet Onboard can be configured to automatically ensure users have their wallet setup to use Polygon.
```
Comment (ashucoder9) [inline docs/develop/wallets/plaid-wallet-onboard.md:19]:
```suggestion
Wallet Onboard can be configured to automatically ensure that users have their wallets set up to use the Polygon network.
```

File: docs/develop/wallets/plaid-wallet-onboard.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+<div align="center">
+<img width="603" src={useBaseUrl("img/plaid-wallet-onboard/splash.png")} />
+</div>
+
+## Installation and Prerequisites
+
+Wallet Onboard can be used in any JavaScript web dApp, and there is a [React library](https://github.com/plaid/react-plaid-link#usage-with-wallet-onboard) for React applications as well.
```
Comment (ashucoder9) [inline docs/develop/wallets/plaid-wallet-onboard.md:27]:
```suggestion
Wallet Onboard can be used in any JavaScript-based web dApp, and there is a [React library](https://github.com/plaid/react-plaid-link#usage-with-wallet-onboard) for ReactJS applications as well.
```

File: docs/develop/wallets/plaid-wallet-onboard.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+</script>
+```
+
+If a user connects their wallet and it is not using Polygon, Wallet Onboard will attempt to add or switch to Polygon automatically, or will instruct the user to switch.
+
+<div align="center">
+<img width="244" src={useBaseUrl("img/plaid-wallet-onboard/switch.png")} />
+</div>
+
+For a complete overview of the configuration options and capabilities of Wallet Onboard, refer to [its docs](https://plaid.com/docs/wallet-onboard/).
```
Comment (ashucoder9) [inline docs/develop/wallets/plaid-wallet-onboard.md:64]:
```suggestion
For a complete overview of the configuration options and capabilities of Wallet Onboard, please refer to the [official Wallet Onboard documentation](https://plaid.com/docs/wallet-onboard/).
```

**Comment by ashucoder9** [comment]
hey @clayallsopp, I guess this PR is still a WIP. Thanks for the contribution and suggested a few changes.

**Comment by clayallsopp** [comment]
Great suggestions @ashucoder9 - any idea why the preview builds are failing? Or is that expected?

**Comment by ashucoder9** [comment]
That is probably due to the Algolia Search API key. We have contacted the DevOps team to fix it asap.

**Comment by ashucoder9** [comment]
Just noticed that the security checks are passing now ✨

**Comment by ashucoder9** [comment]
Fleek-based builds are just meant for previews. We are planning to upgrade our plan to accommodate builds for new PR in the future.

**Comment by clayallsopp** [comment]
@ashucoder9 alrighty! I marked this as ready for review

**Comment by ashucoder9** [comment]
Thanks @clayallsopp!

**Review by ashucoder9 — APPROVED** [review]

**Pull request merged**